<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FSG Missions Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../missions_editor.qrc">
    <normaloff>:/images/missions.svg</normaloff>:/images/missions.svg</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="WidgetDoc" name="widgetDoc" native="true">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="WidgetCGI" name="widgetCGI" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecentFiles">
     <property name="title">
      <string>Recent files</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecentFiles"/>
    <addaction name="actionClearRecent"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewOrbit"/>
    <addaction name="actionViewWorld"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save (Ctrl+S)</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionViewOrbit">
   <property name="text">
    <string>Orbit</string>
   </property>
  </action>
  <action name="actionViewWorld">
   <property name="text">
    <string>World</string>
   </property>
  </action>
  <action name="actionClearRecent">
   <property name="text">
    <string>Clear Recent</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WidgetCGI</class>
   <extends>QWidget</extends>
   <header>gui/WidgetCGI.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetDoc</class>
   <extends>QWidget</extends>
   <header>gui/WidgetDoc.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../missions_editor.qrc"/>
 </resources>
 <connections/>
</ui>
